ics-attack/relationship/relationship--90818d25-6ece-4035-aece-62e489abef7d.json

```diff
@@ -5,7 +5,7 @@
     "objects": [
         {
             "type": "relationship",
-            "target_ref": "attack-pattern--ab390887-afc0-4715-826d-b1b167d522ae",
```

Comment:
This relationship was actually a duplicate and we're going to remove it in #122. We're actually using #122 to supercede this PR since it removes the errant relationship instead of retargeting it, so I'm closing this PR.